draft-ietf-tls-extended-key-update.md

```diff
@@ -512,13 +512,106 @@ Auth | {CertificateVerify}
 
 #  DTLS 1.3 Considerations
 
+Unlike TLS 1.3, DTLS 1.3 implementations must take into account that handshake
```

Comment:
This section needs to be updated to align with https://github.com/tlswg/tls-key-update/pull/40.

Reply:
Definitely.

Before I do that I need to merge this PR, then PR 40 and finally update this section again to avoid merge conflicts.

Reply:
why not publish PR 40 first and then revise this PR ?